src/server.c

```diff
@@ -97,6 +99,50 @@ static int avoid_dupes(const char *url)
         return retval;
 }
 
+/*
+ * Read URLs from environment variable, space separated
+ */
+int configure_urls(void)
+{
+        const char *env_var = getenv("CLR_DEBUGINFO_URLS");
+        const char *token = env_var;
+        int count = 0;
+        char **urls_new = NULL;
+        if (env_var) while (1) {
```

Comment:
code style: An `if` block must always be enclosed with curly brackets. Also, please move the `while (1) ...` to a new line.
```
if (env_var) {
        while (1) {
...
```